src/deepsparse/pipeline.py

```diff
@@ -539,8 +568,25 @@ def _run(self, *args, **kwargs):
             )
 
         # run pipeline
-        engine_inputs: List[numpy.ndarray] = self.process_inputs(pipeline_inputs)
+        if self.use_dynamic_batch():
+            return self._run_with_dynamic_batch(pipeline_inputs)
```

Comment:
previously
```
pipeline_inputs = self.parse_inputs(*args, **kwargs)

if self.use_dynamic_batch():
    return self._run_with_dynamic_batch(pipeline_inputs)

def _run_with_dynamic_batch(self, pipeline_inputs: BaseModel) -> BaseModel:
        pipeline_inputs = pipeline_inputs.split()
```

suggestion
```
pipeline_inputs = self.parse_inputs(*args, **kwargs)
if isinstance(pipeline_inputs, Tuple):
    pipeline_inputs, split_kwargs = pipeline_inputs
else:
    split_kwargs = {}

if self.use_dynamic_batch():
    return self._run_with_dynamic_batch(pipeline_inputs, split_kwargs)

def _run_with_dynamic_batch(self, pipeline_inputs: BaseModel, split_kwargs: Dict[str, Any]) -> BaseModel:
        pipeline_inputs = pipeline_inputs.split(**split_kwargs)
```

Reply:
This seems like a more intuitive way of passing split arguments without have to override `_run_with_dynamic_batch`

Reply:
Accepted!